web-ui/src/components/reviews/periods/ReviewPeriods.jsx

```diff
@@ -409,41 +425,29 @@ const ReviewPeriods = ({ onPeriodSelected, mode }) => {
           periods
             .sort((a, b) => {
               return a.reviewStatus === b.reviewStatus
-                ? ('' + a.name).localeCompare(b.name)
+                ? (a.name || '').localeCompare(b.name)
                 : a.reviewStatus === ReviewStatus.OPEN
                   ? -1
                   : 1;
             })
-            .map(
-              (
-                {
-                  name,
-                  reviewStatus,
-                  id,
-                  launchDate,
```

Comment:
The next three properties are no longer used. Removing them causes several lines to be reformatted by Prettier.